fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsCache.java

```diff
@@ -194,6 +203,9 @@ public Optional<ColumnStatistic> join() {
                         return Optional.of(c);
                     }
                 };
+                if (c == ColumnStatistic.UNKNOWN) {
```

Comment:
I think we should not save Unknown stats into statsTable.

Reply:
why

Reply:
There're too many reasons to cause it be unknown even it actually exists, I think we should passively handle unknown

Reply:
Ok, I got your idea.